S23/thofstrand/lowerCase.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+
+
+
+def lowerCase(word, lang):
+    word = word.strip()
+    lang = lang.split('-')
+    langStart = lang[0]
+
+    if len(langStart) != 2:
+        return "Invalid Language"
+    
+    if "en" == langStart:
+        lower_word = word.lower()
+        return lower_word
+    
+    if "tr" == langStart or 'az' == langStart:
+        
+        if "I" in word:
+            word_list = [*word]
+            indices = [i for i, x in enumerate(word_list) if x == "I"]
+            
+            for x in indices:
+                word_list[x] = 'ı'
+
+            lower_word = combineToString(word_list)
+            lower_word = lower_word.lower()
+            return lower_word
+        else:
+            return word
+        
+    if "ga" == langStart:
+        lowCase = word.lower()
+        caps = ['A','E','I','O','U','Á','É','Í','Ó','Ú']
+        if word[0] == 'n' or word[0] == 't':
+            if len(word) > 2:
+                if word[1] in caps and word[2].isalpha(): #must check isalpha as Õ parses as `O`,`~`
+                    word_list2 = [*lowCase]
+                    word_list2.insert(1,'-')
+                    lower_word = combineToString(word_list2)
+                    return lower_word
+                else:
+                    word = word.lower()
+                    return word
+            else:
+                if word[1] in caps: #must check isalpha as Õ parses as `O`,`~`
+                    word_list2 = [*lowCase]
+                    word_list2.insert(1,'-')
+                    lower_word = combineToString(word_list2)
+                    return lower_word
+                else:
+                    word = word.lower()
+                    return word
+
+            
+        else:
+                word = word.lower()
+                return word
+        
+    if "el" == langStart:
+        if word[-1] == 'Σ':
+            lower_word = word.lower()
+            lower_word_list = [*lower_word]
+            lower_word_list[-1] = 'ς'
+            lower_word = combineToString(lower_word_list)
+            return lower_word
+    else:
```

Comment:
You should have many more test cases for other languages like Spanish, French, etc. that are handled like English.  These tests will fail given the current code.